src/scss/layout/_header.scss

```diff
@@ -1,3 +1,61 @@
-.header {
-  height: 90px;
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
+
+body{
+    max-width: 1440px;
+    margin: auto;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 80px;
+    background-color: var(--white);
+    padding: 31px 80px 31px 90px;
+}
+
+li, a, .btn {
```

Comment:
create classes for these elements using BEM notation.
When using tags as selectors, styles could affect other elements in other components.